### ISSUE TITLE
Vertical Navigation: selected link within the list item needs aria-current attribute added

### ISSUE THREAD

**Issue by SparrowSpit** [CMSgov/design-system#1979]
**Describe the bug**
The [MDCT Team alerted the team](https://cmsgov.slack.com/archives/CHH0381RD/p1658248622172569) that the [vertical navigation component](https://design.cms.gov/components/vertical-navigation) would be improved by adding the [aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) attribute to the currently selected link.

**To Reproduce**
1. Go to [CMS DS Vertical Navigation component](https://design.cms.gov/components/vertical-navigation/)
2. Open browser dev tools
3. Inspect the currently selected list item
4. You will notice that there is no aria-current attribute applied

**Expected behavior**
As a screen reader user I would expect to be able to hear which item from the list is currently selected. 

**Desktop**
 - OS: Mac
 - Browser: Chrome
 - Version: 103.0.5060.114

**Additional context**
Info regarding aria-current:
- The aria-current attribute indicates the element on which it is set
- Only mark one element in a set of elements as current with aria-current.
- The aria-current attribute accepts a limited list of [values](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#values) including page, step, location, date, time, true, and false. I think to avoid confusion we should limit this to "true" as the given value. I'm open to discussion on this but hoping to help developers avoid having to decide this on their own.

The output html should look something like this with aria-current added to the `<a>`:
`<li class="ds-c-vertical-nav__item"><a class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current special-link" href="javascript:void(0);" aria-current="true">Grandchild link</a></li>`
